babel/dates.py

```diff
@@ -1677,6 +1675,33 @@ def get_day_of_year(self, date: datetime.date | None = None) -> int:
             date = self.value
         return (date - date.replace(month=1, day=1)).days + 1
 
+    def get_week_of_year(self, value: datetime.datetime | datetime.date | None = None) -> int:
```

Comment:
Should these methods be public or do we want to keep them private? Either way, it might be nice to add a docstring and some examples perhaps?

Reply:
I think these methods should be kept private.

If there is a use case that determines week of year/month from a date/datetime instance, we should add them as helper methods to babel/dates.py, not instance methods of `DateTimeFormat`.